src/components/Nav/index.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 // eslint-disable-next-line no-unused-vars
-import { BrowserRouter as Router, Link } from 'react-router-dom';
+import { Link } from 'react-router-dom';
```

Comment:
Is this still working ? I'm just mentioning it cause in the past, I needed `Router` obj in this context to make `Link` work.

Reply:
Yes. Still working

Reply:
Wonder why... Anyway ty : )